calcom/cal.com
# chore: preserve non-org dashboard for platform users

## What does this PR do?

Preserve regular dashboard for platform users since they don't have custom domains



https://github.com/user-attachments/assets/be33f51e-b6c6-4db0-985a-e02321b36829


In order to preserve the dashboard with minimal change, we try to exclude the platform org profiles when it's not necessary

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- create a platform org
- try to access public profile page
- it should still point to /username
- /platform/settings should be accessible
